CHANGELOG.md

```diff
@@ -52,9 +52,7 @@ Released on Dec 11, 2019.
 
 ### Enhancements
 
-- Add convenience `parents()` and `children()` classmethods to all State objects for navigating the hierarchy - [#1784](https://github.com/PrefectHQ/prefect/pull/1784)
```

Comment:
Looks like you're removing a valid entry here

Reply:
I think it was actually doubled in the original PR then - it's here a few lines down https://github.com/PrefectHQ/prefect/pull/1854#diff-4ac32a78649ca5bdd8e0ba38b7006a1eL59

Reply:
Ah!